frontend/public/components/dashboard/events-card/events-card.scss

```diff
@@ -16,7 +16,7 @@
 
 .co-events-card__item {
   border-bottom: solid 1px $pf-color-black-200;
-  padding: 0 1.5em;
+  padding: 0 1.5em 1.5em 1.5em;
```

Comment:
nitpic but maybe convert this to REM just for consistency.

Reply:
Converted to rem for left/right but changed bottom to a static `15px` to make sure it's the same as the top padding applied to `co-sysevent--transition`. We could definitely improve spacing overall in a future pass maybe - right now we seem to use a mix of `rem`, `em` and `px` everywhere.

![image](https://user-images.githubusercontent.com/9122899/61309185-b6dc6500-a7bf-11e9-93c6-9b6e3f696a55.png)

Reply:
yes, I agree. There's opportunity to scrub these .scss files and align things. We could do more to ensure a proper PF4 layout as well.